src/opnsense/www/js/opnsense.js

```diff
@@ -135,26 +140,27 @@ function setFormData(parent,data) {
                         targetNode.empty(); // flush
                         $.each(node[keypart],function(indxItem, keyItem){
                             if (keyItem["selected"] != "0") {
-                                targetNode.append("<option value='"+indxItem+"' selected>" + keyItem["value"] + " </option>");
+                                targetNode.append(`<option value="${indxItem}" selected>${keyItem["value"]}</option>`);
                             } else {
-                                targetNode.append("<option value='"+indxItem+"'>" + keyItem["value"] + " </option>");
+                                targetNode.append(`<option value="${indxItem}">${keyItem["value"]}</option>`);
```

Comment:
@fichtner this is not supported by IE11 but by Edge, Chome and Firefox. should I undo this two lines?

Reply:
this is standard but it seems like IE does not get modern features anymore. If you look at this page, you see that IE 11 does not get modern JS features anymore:
https://kangax.github.io/compat-table/es6/

Reply:
done btw.